File: fix_types.js
const fs = require('fs');
let content = fs.readFileSync('src/types.ts', 'utf8');

content = content.replace("export export type GamePhase = 'TRANSFERS' | \n  | 'CREATION_BASIC_INFO'", "export type GamePhase = 'TRANSFERS'\n  | 'CREATION_BASIC_INFO'");
content = content.replace("export export type GamePhase = 'TRANSFERS' |", "export type GamePhase = 'TRANSFERS'\n ");
content = content.replace(" | 'CREATION_BASIC_INFO'", " | 'CREATION_BASIC_INFO'"); // just in case

fs.writeFileSync('src/types.ts', content);

File: fix_types.py
import re

with open('src/core/domain/draftEngine.ts', 'r') as f:
    content = f.read()

content = content.replace("export function applyDraftResultToPlayer(player: PlayerState, result: DraftResult, dna: PlayerDNA[]): PlayerState {", "import { PlayerState } from '../../types';\n\nexport function applyDraftResultToPlayer(player: PlayerState, result: DraftResult, dna: PlayerDNA[]): PlayerState {")

with open('src/core/domain/draftEngine.ts', 'w') as f:
    f.write(content)
